test/test_nn.py

```diff
@@ -3790,6 +3803,48 @@ def _test_batchnorm_eval(self, test_type=torch.FloatTensor):
         self.assertEqual(res1, res2)
         self.assertEqual(grad1, grad2)
 
+    def _test_batchnorm_simple_average(self, test_type=torch.FloatTensor):
+        module = nn.BatchNorm1d(3, momentum=None).type(test_type)
+        zeros = torch.zeros(3).type(test_type)
+        ones = torch.ones(3).type(test_type)
+        self.assertEqual(module.running_mean, zeros)
+        self.assertEqual(module.running_var, ones)
+
+        data1 = Variable(torch.rand(4, 3).type(test_type), requires_grad=True)
```

Comment:
nit: `Variable` wrapper is not needed anymore. We are also removing it from other test cases, so could you also not use it?